frontend/ui/widget/radiobuttontable.lua

```diff
@@ -52,17 +52,18 @@ function RadioButtonTable:init()
         local horizontal_group = HorizontalGroup:new{}
         local row = self.radio_buttons[i]
         local column_cnt = #row
-        local sizer_space = self.sep_width * (column_cnt - 1) + 2
+        local sizer_space = (self.sep_width + 2 * self.padding) * (column_cnt - 1)
```

Comment:
Is that 2 and 1 supposed to be raw?

Reply:
It is a space between buttons in a row.
Currently there aren't any multi-column radio button tables.

Reply:
Yes, but is it scaleBySized down the line?

Reply:
You only need one scaleBySize()d thingy (self.padding and self.sep_width are), the other stuff like 2 or -1 are involved in multiplication of these, so it's fine (you don't want to multiply 2 scaleBySize things :)
(But there was a but before with the + 2 at end.)

Reply:
There aren't any raw dimensions in the new code.

Reply:
> (But there was a bug before with the + 2 at end.)

I was going to say — no it freaking wasn't! lol

Reply:
> there was a bug before with the + 2 at end

https://github.com/koreader/koreader/blob/5e224d541ff6023247aa24c4cb039dbe9df32c97/frontend/ui/widget/buttontable.lua#L51